### PR TITLE
Link to component package and revision

### LINKED ISSUE #534: need link to "provider" in full details page
When looking at a full details page like  https://clearlydefined.io/definitions/npm/npmjs/-/gulp-remote-src-vscode/0.5.1 there is no way to get to the actual component on its native site (e.g., npmjs.com).

### PR DESCRIPTION
**Pull request by storrisi**
Fixes #534 

To improve consistency, i've created a reusable component, used both in lists and in the details page.

### CODE REVIEW

File: src/components/Navigation/Sections/HeaderSection.js [hunk 1]
```diff
@@ -4,11 +4,14 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { Row, Button, Col } from 'react-bootstrap'
 import isEmpty from 'lodash/isEmpty'
+import get from 'lodash/get'
```
Comment (dabutvin) [inline src/components/Navigation/Sections/HeaderSection.js]: should not need to import this here

**Comment by storrisi** [comment]
Thanks @dabutvin , i've removed unused libs here

**Comment by storrisi** [comment]
I think this one is ready to be merged, @dabutvin what do you think?

**Comment by dabutvin** [comment]
👍

**Pull request merged**